lib/logstash/agent.rb

```diff
@@ -48,6 +48,10 @@ class LogStash::Agent < Clamp::Command
     I18n.t("logstash.agent.flag.configtest"),
     :attribute_name => :config_test
 
+  option ["--auto-download"], :flag,
+    I18n.t("logstash.agent.flag.autodownload"),
+    :attribute_name => :auto_download
```

Comment:
Don't need this attribute_name since the default name will be the same as your flag (`--auto-download` becomes `:auto_download`).

Also, let's set `:default => false` to make it explicit (otherwise the default is nil)